FrontEnd/src/components/authorization/LoginContent.jsx

```diff
@@ -138,35 +128,33 @@ const LoginContent = (props) => {
                     value={enteredPassword}
                     onChange={passwordChangeHandler}
                   />
-                  <button onClick={passwordVisisbilityHandler} type="button">
+                  <span className={classes["password-visibility"]} onClick={passwordVisisbilityHandler}>
```

Comment:
Why did change `button` to `span`?

Reply:
It's only an icon element for handling password visibility. It's more convenient to style this element inside of input element. And I've investigated such elements on the Internet, most of them were added as span element.